wpilibc/athena/src/Task.cpp

```diff
@@ -51,7 +51,7 @@ std::thread::native_handle_type Task::native_handle() {
  * @return true on success.
  */
 bool Task::Verify() {
-  TASK id = (TASK)m_thread.native_handle();
+  TASK id = reinterpret_cast<TASK>(m_thread.native_handle());
```

Comment:
VxWorks TASK_ID I think.  But std::thread wasn't available anyway, so this code is completely different.